docs/source/conf.py

```diff
@@ -413,7 +406,7 @@ def get_members(obj, typ, public_only=True):
             if public_only and name.startswith('_'):
                 continue
             try:
-                documenter = get_documenter(safe_getattr(obj, name), obj)
+                documenter = get_documenter(None, safe_getattr(obj, name), obj)
```

Comment:
What is this change fore?

Reply:
AFAICT, there's no behavioral change here. Sphinx v1.7.0 has changed the function signature of `get_documenter`. It now requires three positional args:

https://github.com/sphinx-doc/sphinx/blob/5558b63315438cfaba1d6f45e1ce59da8551f94a/sphinx/ext/autosummary/__init__.py#L163

...compared to < v1.7.0:

https://github.com/sphinx-doc/sphinx/blob/645044d7f03c7fcb4c39a8edf40aa66f6d6c2b64/sphinx/ext/autosummary/__init__.py#L160

If we set the new arg `app` as `None`, it will have the same effect as < v1.7.0. `app` is passed into `get_documenters` (note it's another function):

https://github.com/sphinx-doc/sphinx/blob/5558b63315438cfaba1d6f45e1ce59da8551f94a/sphinx/ext/autosummary/__init__.py#L190

https://github.com/sphinx-doc/sphinx/blob/5558b63315438cfaba1d6f45e1ce59da8551f94a/sphinx/ext/autodoc/__init__.py#L1491-L1497

...compared to < v1.7.0:

https://github.com/sphinx-doc/sphinx/blob/645044d7f03c7fcb4c39a8edf40aa66f6d6c2b64/sphinx/ext/autosummary/__init__.py#L188